### PR TITLE
FIX travis

### PR DESCRIPTION
**Pull request by angelmoya**
revert 

div[@hasclass('o_sub_menu')]

### CODE REVIEW

**Comment by angelmoya** [comment]
Hi @pedrobaeza 

how can I see travis?

I revert last change that was propossed by @lasley 
```
div[@hasclass('o_sub_menu')]
```
I do not know if that is the problem

**Comment by pedrobaeza** [comment]
Uhm, Travis should be triggered automatically. Let's wait if there's some stuck.

**Comment by lasley** [comment]
That's weird - I don't see a queued build...

**Comment by pedrobaeza** [comment]
Let's see if now it's triggered...

**Comment by pedrobaeza** [comment]
Nop. We should try tomorrow or open a new PR.

File: web_responsive/views/web.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
               name="App Drawer - Web Client"
               >
 
-        <xpath expr="//div[@contains('class', 'o_sub_menu')]" position="replace" />
```
Comment (lasley) [inline web_responsive/views/web.xml:15]: Sorry I think my suggestion was misunderstood - I should have caught it too, my bad. This is what we want:

```
//div[hasclass('o_sub_menu')]
```

[Examples](https://github.com/odoo/odoo/search?utf8=%E2%9C%93&q=%22hasclass%28%22&type=)

**Comment by angelmoya** [comment]
@pedrobaeza @lasley 

I fixed hasclass tag.... I tested local, but travis and runbot are not working...

**Comment by pedrobaeza** [comment]
Let's see if it works now. Any way, branch is red, so if @lasley thinks the change is correct, just merge.

**Comment by lasley** [comment]
I'm going to double check this using my Runbot and assuming we're good functionally, just merging is exactly the plan!

**Comment by lasley** [comment]
https://github.com/PESOL/web/pull/1

**Comment by angelmoya** [comment]
Sorry for this errors @lasley ... I tried to improve xml, but sometimes is better stop and do it slowly

**Comment by lasley** [comment]
Quick merged, as discussed. I squashed our commits together because it was cleaner IMO

**Pull request merged**